wlauto/utils/fps.py

```diff
@@ -19,8 +19,6 @@ class FpsProcessor(object):
     Provide common object for processing surfaceFlinger output for frame
     statistics.
 
-    This processor adds four metrics to the results:
```

Comment:
Rather than removing it I would change it to say that it returns the values below